scale-tests/run_spark_scale_test.sh

```diff
@@ -0,0 +1,226 @@
+
+## Deploy streaming jobs
+### Deploy failing jobs
+KAFKA_JAR=${KAFKA_JAR:-http://infinity-artifacts.s3.amazonaws.com/scale-tests/dcos-spark-scala-tests-assembly-20180523-c450832.jar}
+SUBMISSIONS_FINITE_OUTPUT_FILE="${TEST_NAME}-finite-submissions.out"
+SUBMISSIONS_INFINITE_OUTPUT_FILE="${TEST_NAME}-infinite-submissions.out"
+PRODUCER_SPARK_CORES_MAX=2
+PRODUCER_SPARK_EXECUTOR_CORES=2
+CONSUMER_BATCH_SIZE_SECONDS=10
+CONSUMER_SPARK_CORES_MAX=1
+CONSUMER_SPARK_EXECUTOR_CORES=1
+
+./scale-tests/kafka_cassandra_streaming_test.py \
+  $DISPATCHERS_FINITE_JSON_FILE \
+  $INFRASTRUCTURE_JSON_FILE \
+  $SUBMISSIONS_FINITE_OUTPUT_FILE \
+  --jar $KAFKA_JAR \
+  --num-producers-per-kafka $NUM_PRODUCERS_PER_FINITE_KAFKA \
+  --num-consumers-per-producer $NUM_CONSUMERS_PER_FINITE_PRODUCER \
+  --producer-number-of-words $NUM_PRODUCER_WORDS \
+  --producer-words-per-second $NUM_PRODUCER_WORDS_PER_SECOND \
+  --producer-spark-cores-max $PRODUCER_SPARK_CORES_MAX \
+  --producer-spark-executor-cores $PRODUCER_SPARK_EXECUTOR_CORES \
+  --producer-must-fail \
+  --consumer-must-fail \
+  --consumer-write-to-cassandra \
+  --consumer-batch-size-seconds $CONSUMER_BATCH_SIZE_SECONDS \
+  --consumer-spark-cores-max $CONSUMER_SPARK_CORES_MAX \
+  --consumer-spark-executor-cores $CONSUMER_SPARK_EXECUTOR_CORES
+
+### Deploy finite jobs
+./scale-tests/kafka_cassandra_streaming_test.py \
+  $DISPATCHERS_FINITE_JSON_FILE \
+  $INFRASTRUCTURE_JSON_FILE \
+  $SUBMISSIONS_FINITE_JSON_FILE \
+  --jar $KAFKA_JAR \
+  --num-producers-per-kafka $NUM_PRODUCERS_PER_FINITE_KAFKA \
+  --num-consumers-per-producer $NUM_CONSUMERS_PER_FINITE_PRODUCER \
+  --producer-number-of-words $NUM_PRODUCER_WORDS \
+  --producer-words-per-second $NUM_PRODUCER_WORDS_PER_SECOND \
+  --producer-spark-cores-max $PRODUCER_SPARK_CORES_MAX \
+  --producer-spark-executor-cores $PRODUCER_SPARK_EXECUTOR_CORES \
+  --consumer-write-to-cassandra \
+  --consumer-batch-size-seconds $CONSUMER_BATCH_SIZE_SECONDS \
+  --consumer-spark-cores-max $CONSUMER_SPARK_CORES_MAX \
+  --consumer-spark-executor-cores $CONSUMER_SPARK_EXECUTOR_CORES
+    
+### Deploy infinite jobs
+INFINITE_PRODUCER_NUMBER_OF_WORDS=0
+
+./scale-tests/kafka_cassandra_streaming_test.py \
+  $DISPATCHERS_INFINITE_JSON_FILE \
+  $INFRASTRUCTURE_JSON_FILE \
+  $SUBMISSIONS_FINITE_JSON_FILE \
+  --jar $KAFKA_JAR \
+  --num-producers-per-kafka $NUM_PRODUCERS_PER_INFINITE_KAFKA \
+  --num-consumers-per-producer $NUM_CONSUMERS_PER_INFINITE_PRODUCER \
+  --producer-number-of-words $INFINITE_PRODUCER_NUMBER_OF_WORDS \
+  --producer-words-per-second $NUM_PRODUCER_WORDS_PER_SECOND \
+  --producer-spark-cores-max $PRODUCER_SPARK_CORES_MAX \
+  --producer-spark-executor-cores $PRODUCER_SPARK_EXECUTOR_CORES \
+  --consumer-batch-size-seconds $CONSUMER_BATCH_SIZE_SECONDS \
+  --consumer-spark-cores-max $CONSUMER_SPARK_CORES_MAX \
+  --consumer-spark-executor-cores $CONSUMER_SPARK_EXECUTOR_CORES
+
+## Deploy GPU jobs
+git checkout st-gpu-strict
```

Comment:
Be careful changing this mid-script. Rather ensure that the branch we are running is consistent.

Ideally, for scripts such as this, we should *only* run from `master` (or some "stable" branch).